### ISSUE TITLE
Dynamic project path resolution for monorepo

### ISSUE THREAD

**Issue by heavenkiller2018** [firsttris/vscode-jest-runner#89]
there are many projects which have their own jest configuration in yarn monorepo workspace, how to configure to make sure jest runner run in every projects with its coreect jest configuration?

**Comment by firsttris** [comment]
one possible solution would be to add those folders/projects of your monorepo to you workspace using the "add folder to workspace" option. 

then vscode & jest would recognize those as individual projects.

**Comment by firsttris** [comment]
generally if you have a monorepo which has jest and its config installed to <MonoRepoRoot> it should works out of the box. 

but if you want's it to use different jest config for each project, there is no way how my plugin can tell if a folder is actually project in a monorepo.

well there is one way - if you use the workspace option of vscode.

there are serval other ideas:
https://github.com/firsttris/vscode-jest-runner/issues/68

and also a PR
https://github.com/firsttris/vscode-jest-runner/pull/84 (
with this PR it would be possible to define one specific project-folder, instead of a dynamic project folder resolution in a mono repo, which i think i what you requested.

**Comment by heavenkiller2018** [comment]
'add those folders/projects of your monorepo to you workspace using the "add folder to workspace" option.'  may be not fit for me, because some others config may be effected if one folder of yarn monorepo was configured to vscode's workspace. 

`there is no way how my plugin can tell if a folder is actually project in a monorepo.` :
myabe you could resolve package.json file in workspaceroot, such as 

```json
{
  "name": "monoapp",
  "version": "1.0.0",
  "private": true,
  "main": "index.js",
  "workspaces": {
    "packages": [
      "apps/*",
      "packages/*"
    ]
  },
```


the value of 'private' and 'workspaces' should show you which folder is actually a project. and then you can look into package.json of the correspondent project to do further steps. It would be wonderful if each project in yarn monorepo can't be configured separately in jest-runner!

**Comment by firsttris** [comment]
in version 0.4.24 there is a option to define a fixed project path from where jest & config is used.

**Comment by floratmin** [comment]
I have the same problem with nx workspace. When i specify the path to a jest.config.js of one library, i can run the tests of this library without problem, but tests in other libraries do not run. When I do not specify jest.config.js it runs the test two times, one time passing and one time failing.

**Comment by firsttris** [comment]
Dynamic Jest config (jest.config.js) resolution is now available in version 0.4.27 thx to @vrptnc 

please test & report back
